CMakeLists.txt

```diff
@@ -105,52 +118,77 @@ else(HAVE_ICONV_CONST)
   remove_definitions(-DHAVE_ICONV_CONST)
 endif(HAVE_ICONV_CONST)
 
-target_include_directories(tinygettext PUBLIC ${ICONV_INCLUDE_DIR})
+include_directories(${ICONV_INCLUDE_DIR})
 target_link_libraries(tinygettext PUBLIC ${ICONV_LIBRARY})
 
-endif(HAVE_SDL)
+
 
 ## Debug options
 
 option(WERROR "Stops on first compiler warning in debug mode" OFF)
 option(WARNINGS "Enable long list of warnings for compiler to check" ON)
 
-add_definitions(-std=c++0x -O3 -Wall -Wextra -Weffc++ -pedantic)
+if ("${CMAKE_VERSION}" VERSION_GREATER 3.0.99)
+  set_property(GLOBAL PROPERTY CXX_STANDARD 11)
+else ("${CMAKE_VERSION}" VERSION_GREATER 3.0.99)
+  # http://www.guyrutenberg.com/2014/01/05/enabling-c11-c0x-in-cmake/
+  include(CheckCXXCompilerFlag)
+  CHECK_CXX_COMPILER_FLAG("-std=c++11" COMPILER_SUPPORTS_CXX11)
+  if(COMPILER_SUPPORTS_CXX11)
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
```

Comment:
I think just setting  cmake_minimum_required to 3.1 would work too; Travis is using CMake 3.3, and any decent development system will have a recent CMake.